guard/src/commands/completions.rs

```diff
@@ -23,44 +19,19 @@ impl From<String> for Shell {
     }
 }
 
-#[derive(Default, Debug)]
-pub struct Completions {}
-
-const SHELL_TYPES: [&str; 3] = ["bash", "zsh", "fish"];
-const SHELL: (&str, char) = ("shell", 's');
-
-impl Command for Completions {
-    fn name(&self) -> &'static str {
-        COMPLETIONS
-    }
-
-    fn command(&self) -> clap::Command {
-        clap::Command::new(COMPLETIONS)
-            .about("Generate auto-completions for all the sub-commands in shell.")
```

Comment:
Do we not care that we're losing this `about`?

Reply:
good catch, this was just a mistake on my part when migrating